src/types.ts

```diff
@@ -3,7 +3,19 @@
  * Partial implementation of RequestInit with the addition of "data" property which value will be returned from the mock.
  */
 export type MockOptions = {
+    /** @deprecated use response.data */
```

Comment:
I don't feel confidend deprecating the root data property and using the `response.data` instead.
It's logically correct to have the data inside the response object, as it de facto mocks the response body, but it also gets rid of the simplicity of `mock('..', { data: {...} })`.
On the other hand, keeping both seems unnecessary.

Reply:
I don't think having two ways to do such a same thing is a good idea, I find it unnecessary and confusing. I would suggest deprecating it.